bin/kafka-streams-application-reset.sh

```diff
@@ -18,4 +18,4 @@ if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
 
-exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.tools.StreamsResetter "$@"
+exec $(dirname $0)/kafka-run-class.sh kafka.tools.StreamsResetter "$@"
```

Comment:
It's not great using that package name in `core`, but maybe that's the least bad option. If we do that, we should add some class-level documentation explaining why it is like that. I have to say that having "API" discussions in a bug fix release after the RC has gone out is not ideal. Are we sure about the package name? Do we need to add a warning to the documentation saying that the name may change?

The other question is what we are doing in `trunk` until `KIP-4` comes around. We should probably do whatever we're doing in the `0.10.0` branch for consistency.